packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -31,7 +31,9 @@ const pkg = require('@fluentui/react-northstar/package.json');
 const componentMenu: ComponentMenuItem[] = _.sortBy(componentInfoContext.parents, 'displayName');
 const behaviorMenu: ComponentMenuItem[] = require('../../behaviorMenu');
 
-const componentsBlackList = ['Debug', 'Design', 'Datepicker'];
+const PROD = process.env.NODE_ENV === 'production';
```

Comment:
There is no sense to embed this condition. I am usually using search by `process.env.NODE_ENV` if there is something conditional

Reply:
We were reusing this condition in 2 places in this file, I would be introducing the 3rd. Should it always be "in place"?